### DISCUSSION TITLE
Monitor EMQX Authorization queue

### DISCUSSION THREAD

**Discussion by ddellarocca** [emqx/emqx#12718] [Q&A]
Hi,
This discussion is related to https://github.com/emqx/emqx/discussions/12618, in particular apart from CPU throttling is there a way to know if the authorization is lagging behind the message throughput? The key point here is that I would like to check if enough resources are assigned to EMQX in order to validate message spikes within a time threshold.
For example if in an EMQX cluster there are 10k clients connected to it and all of them send a message at the same time generating a 10k msg peak i would like to know if all of those messages gets validated under 1 second.
I've looked at the Prometheus metrics exposed by EMQX but I didn't find any that fits my needs, is there a documentation page where is it possible to find all the metrics exposed and their meaning?
Thanks.

**Comment by ieQu1** [comment]
Hello,

From a brief look at the code, I don't see any metrics exported by the http auth backend. However I think this is reasonable requirement. Please feel free to submit a feature request.

**Reply by ddellarocca** [reply]
And a generic one for the validation queue? Like also for redis?
